deployment/group_vars/wetc.yml

```diff
@@ -1,24 +1,41 @@
 ## General settings
 BRIDGE_MODE: "NATIVE_TO_ERC"
+HOME_NATIVE_NAME: "ETC"
 
 ## Home contract
 HOME_RPC_URL: "https://ethereumclassic.network"
+HOME_NAME: "Ethereum Classic"
+HOME_WITHOUT_EVENTS: false
 HOME_BRIDGE_ADDRESS: "0x073081832B4Ecdce79d4D6753565c85Ba4b3BeA9"
 HOME_POLLING_INTERVAL: 7000
 
 ## Foreign contract
 FOREIGN_RPC_URL: "https://mainnet.infura.io/"
+FOREIGN_NAME: "Ethereum Mainnet"
+FOREIGN_WITHOUT_EVENTS: false
 FOREIGN_BRIDGE_ADDRESS: "0x0cB781EE62F815bdD9CD4c2210aE8600d43e7040"
 ERC20_TOKEN_ADDRESS: "0x86aaBCc646f290b9Fc9Bd05CE17C3858d1511Da1"
 FOREIGN_POLLING_INTERVAL: 7000
 
-## Gasprice
+## Home Gasprice
 HOME_GAS_PRICE_ORACLE_URL: "https://gasprice-etc.poa.network/"
 HOME_GAS_PRICE_SPEED_TYPE: "standard"
 HOME_GAS_PRICE_FALLBACK: 15000000000      # in wei
 HOME_GAS_PRICE_UPDATE_INTERVAL: 600000
 
+## Foreign Gasprice
 FOREIGN_GAS_PRICE_ORACLE_URL: "https://gasprice.poa.network/"
 FOREIGN_GAS_PRICE_SPEED_TYPE: "standard"
 FOREIGN_GAS_PRICE_FALLBACK: 10000000000   # in wei
 FOREIGN_GAS_PRICE_UPDATE_INTERVAL: 600000
+
+## UI
+UI_TITLE: "TokenBridge UI app - %c"
+UI_DESCRIPTION: "The TokenBridge serves as a method of transferring native tokens from the Ethereum Classic Network to the Ethereum network in a quick and cost-efficient manner."
+UI_PORT: 3001
+UI_HOME_EXPLORER_TX_TEMPLATE: https://blockscout.com/poa/sokol/tx/%s
```

Comment:
the template links should be also updated to reflect the corresponding Blockscout endpoints.

Reply:
Updated.